asmcli/commands/create-mesh.sh

```diff
@@ -154,3 +154,15 @@ prepare_add_to_mesh_environment() {
     organize_kpt_files
   fi
 }
+
+generate_secret_name() {
+  local SECRET_NAME; SECRET_NAME="${1}"
+
+  if [[ "${#SECRET_NAME}" -gt 63 ]]; then
+    local HASH
+    HASH="$(echo "${SECRET_NAME}" | sha256sum | head -c20 || true)"
```

Comment:
very nit: the data is a "digest", the function is a "hash"

Reply:
Done